docs/DEPENDENCY_GRAPH.md

```
@@ -0,0 +1,178 @@
+  IE --> PS
+```
+
+**Intended invariant:** `internal/domain/**` MUST NOT import `internal/repository`, `internal/service`, or `internal/api`.
+
+---
+
+## Upstream vs downstream (runtime perspective)
+
+### Ingest path (metering events)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)